private-templates-service/client/src/components/Dashboard/RecentlyViewed/OwnerAvatar/OwnerAvatar.tsx

```diff
@@ -25,14 +25,14 @@ class OwnerAvatar extends React.Component<Props> {
         personaSize: PersonaSize.size24,
         personas: new Array({ imageUrl: this.props.owner.imageURLs[this.props.index], }),
       };
-      return (<Facepile {...facepileProps} />)
+      return (<Facepile {...facepileProps} ariaLabel={"Facepile displaying profile pictures"} />)
     }
     else if (this.props.owner && this.props.owner.displayNames && this.props.owner.displayNames[this.props.index] && this.props.owner.displayNames[this.props.index][0]) {
       let facepileProps: IFacepileProps = {
         personaSize: PersonaSize.size24,
         personas: new Array({ imageInitials: this.props.owner.displayNames[this.props.index][0], }),
       };
-      return (<Facepile {...facepileProps} />)
+      return (<Facepile {...facepileProps} ariaLabel={"Facepile displaying profile pictures"} />)
```

Comment:
change to 
ariaLabel="Facepile displaying profile pictures"

Reply:
also, what is `facepile`, is it relevant to the user

Reply:
Oh no, I thought we had to include the name of the UI fabric component.

Reply:
I'll change it.